### ISSUE TITLE
Win error and preparations for wx-3.1

### ISSUE THREAD

**Issue by Hakansv** [jongough/ocpn_draw_pi#390]
Jon...
For my test purposes I've built OD for O master v4.99.0. 
1)
VS2017 is somewhat "chinchy" ? Couldn't build wo clarifying:
@@ -649,7 +649,7 @@ bool ODPathPropertiesDialogImpl::UpdateProperties( void )
     ResetGlobalLocale();
     
-    if(m_pPath->m_sTypeString != "EBL" and m_pPath->m_sTypeString != "PIL") SetDialogSize();
+   if(m_pPath->m_sTypeString != "EBL" && m_pPath->m_sTypeString != "PIL") SetDialogSize();
 
2)
What about preparing for coming O v5.0 and wx 3.1: 
I've a lot of warnings like this:
4>c:\builds\ocpn\ocpn_draw_pi\src\odpoint.cpp(480): warning C4996: 'wxPen::wxPen': deprecated: use wxPENSTYLE_XXX constants
4>c:\wxwidgets-3.1.1-rc\include\wx\msw\pen.h(58): note: see declaration of 'wxPen::wxPen'

**Comment by Hakansv** [comment]
Jon..
Sorry for being unclear. I'm referring to branch master_1.5 and use Win10.
And BTW:
When I run it in O 4.990 there seams to be a never ending loop or similar. The whole O's extremely slowed down and and the PC fan is really working hard. All until OD plugin is deactivated.
I'm sorry but can't debug O for the moment. 
Håkan

**Comment by jongough** [comment]
Hi Håkan,
   I have fixed the first issue, it is not something I normally put it, so I don't know what I was thinking! The second one about wx 3.1, I got burned before by being ahead of OCPN so I am using 3.0.4. I will see if I can get a separate environment running with 3.1 to check it out. However, OD using 3.0.4 should work with OCPN 5 using 3.1.

I have not seen the high CPU usage under linux. I will see if I can do a test under windows 7 with VS2017 and see if I get the issue. I don't currently have a win10 system with VS installed, but will see if I can get one soon.

Regards
   Jon
